desc/objectives/linear_objectives.py

```diff
@@ -1388,21 +1383,20 @@ def build(self, use_jit=False, verbose=1):
         eq = self.things[0]
         if self._modes is True:  # all modes
             modes = eq.R_basis.modes
-            idx = np.arange(eq.R_basis.num_modes)
```

Comment:
whats the reasoning for these changes?

Reply:
just simplifying the code to remove unused variables, no change in functionality